lib/street_address.rb

```diff
@@ -867,7 +867,7 @@ def line1(s = "")
           s += " " + street_type2 unless street_type2.nil?
           s += " " + suffix2 unless suffix2.nil?
         else
-          s += number
+          s += number unless number.nil?
```

Comment:
is this the next thing that fails? this makes sense given there's no case for "is a street (no number)", but one side effect is the next few portions may have an extraneous space I guess. i.e. the next lines prepend a space assuming there was always a number starting the string.

Reply:
good catch-- 🤔

Reply:
how do you feel about just adding a `squish` somewhere... maybe just on the return value at the end 🤔

Reply:
ooh-- or `strip` I guess is the right one